scripts/darwin.sh

```diff
@@ -62,5 +66,23 @@ for disk in $DISKS; do
     echo "system: False"
   fi
 
+  partitions="`diskutil list $disk | get_device_identifiers`"
+  partitionmountpoints=""
+
+  for partition in $partitions; do 
+    partitioninfo="`diskutil info $partition`"
+
+    partitionmountpoint=`echo "$partitioninfo" | get_key "Mount Point"`
+
+    if [ "$partitionmountpoint" ]; then
+      partitionmountpoints=$partitionmountpoints,$partitionmountpoint
+    fi
+  done
+  
+  # trim leading ,
+  partitionmountpoints=${partitionmountpoints#,}
+
+  echo "mountpoints: [ $partitionmountpoints ]"
```

Comment:
Lets just expose a comma separated string at this point for consistency with the other operating systems. Happy to accept a PR that parses comma separated lists as arrays in https://github.com/resin-io-modules/drivelist/blob/master/lib/parse.coffee later.